src/Modules/Backend/templates/base/macros.html.twig

```diff
@@ -114,16 +114,16 @@
   </div>
 {% endmacro %}
 
-{% macro emptyState(addItemLink, addItemText, noItemsText, addItemLinkExtraAttributes) %}
-  {% set addItemText = addItemText|default('msg.AddItem'|trans|raw) %}
-  {% set noItemsText = noItemsText|default('msg.NoItems'|trans|raw) %}
+{% macro emptyState(addItemLink, add_item_text, no_items_text, addItemLinkextra_attributes) %}
```

Comment:
Shouldn't `addItemLinkextra_attributes` be changed into `add_item_link_extra_attributes`?

Reply:
fixed 59a1aec83cc9bdfa9656261afafc8ca4e9e78d41